app/models/upload.rb

```diff
@@ -14,6 +14,7 @@ class Upload < ApplicationRecord
   validate :valid_url, if: proc { |upload| upload.url.present? }
   scope :active, -> { where(status: %w[active processed]) }
   scope :archived, -> { where(status: 'archived') }
+  default_scope { order(created_at: :desc) }
```

Comment:
@thatbudakguy it looks like changing the default ordering has some unintended side-effects, like breaking the activity charts on the providers page (https://pod.stanford.edu/organizations):
<img width="1358" alt="Screen Shot 2022-04-27 at 8 05 30 AM" src="https://user-images.githubusercontent.com/458247/165514620-8d7a6de2-5321-46b0-95e3-93553021ed62.png">

There may be other places in the application where changing the default order has unintended side-effects. I'm wondering if it would make sense to go back to setting the order in the `uploads` action instead?

Reply:
dang, I wish we'd had tests for that! It does beg the question of why the charts are relying on `updated_at`, and whether they should be. i'll dig into this more

Reply:
reverted the default ordering so i can focus on other stuff.